activities/creating-partnerships/index.md

```diff
@@ -18,7 +18,6 @@ The Foundation for Public Code is actively looking for diverse public organizati
 
 Only public organizations can become partners.
 Other organizations can become [affiliates](../creating-affiliations/index.md)
-If you're interested in partnership for your organization, or have any questions, please email us at <partnerships@publiccode.net>.
```

Comment:
Because this a page in the staff manual, so this sentence should be moved to a .net page

Or do you think it would be useful to still have these sign posts here and there?

Reply:
It could be rephrased if it is internal facing, but we should still have documentation somewhere on About which email is used for what. 

Also, I don't recall the new design of the website having this info, so perhaps wait with removing it here until there is a plan to actually put it there?

Reply:
Agreed. Until there is a partners page to link to on dot-net, let's phrase it something like, "The email list <partnerships@publiccode.net> is maintained to answer questions and provide information to Organizations interested in partnership." or similar.

Reply:
> Or do you think it would be useful to still have these sign posts here and there?

I do, as that will likely still be useful for us, yet should have a change in tone in the wording.

Reply:
Good points. Sentence restored and reworded.